lib/v8.js

```diff
@@ -15,5 +15,20 @@
 'use strict';
 
 var v8binding = process.binding('v8');
-exports.getHeapStatistics = v8binding.getHeapStatistics;
+var smalloc = require('smalloc');
+
+var data = smalloc.alloc(5, smalloc.Types.Uint32);
```

Comment:
@bnoordhuis This isn't to say I don't like the idea of having a common way to cleanup memory on exit that's been allocated. It is annoying to constantly have to sort through so many extra lines in valgrind. Do you have those documented? Could use that as a starting point and possibly create a class or something that is used for allocations that need to be cleaned up on exit.